### ISSUE TITLE
Pile integrity failure crash

### ISSUE THREAD

**Issue by angband-import** [angband/angband#4225]
**Reported by NickMcConnell on 13 Oct 2019 05:30 UTC**
Yes, this still happens: "I just got the following message on depth level 78, while slogging through the items left behind by a pit of Ainur:

"Pile integrity failure, details written to C:\Games\angband\lib\user\pile_error.txt"." - Pussy Galore

It would be good to get at least better diagnostics for this.

**Comment by NickMcConnell** [comment]
Calling this ongoing as it doesn't seem to be going away, and a better way of dealing with it would be preferable.

**Comment by pete-mack** [comment]
nick--I mentioNed in a closed issue report that the object in question was a monster drop out of player LOS. Surely this helps some?

**Comment by NickMcConnell** [comment]
Yes, possibly - at least we have it mentioned in this ticket now.
